server/app/com/xsn/explorer/services/BlockService.scala

```diff
@@ -502,6 +505,30 @@ class BlockService @Inject() (
     }
   }
 
+  private def getTotalInput(inputs: List[TransactionVIN]): FutureApplicationResult[BigDecimal] = {
```

Comment:
its kind of tested indirectly on the tests for getBlockRewards and since its a private method i did not see the need to test it directly.
would you like me to test it directly?